applets/clock/clock.c

```diff
@@ -987,6 +996,80 @@ position_calendar_popup (ClockData *cd)
 
         gtk_window_move (GTK_WINDOW (cd->calendar_popup), x, y);
         gtk_window_set_gravity (GTK_WINDOW (cd->calendar_popup), gravity);
+    }
+#endif
+    /*Only build wayland support when building in process*/
+#ifdef CLOCK_INPROCESS
+#ifdef HAVE_WAYLAND
+
+    if (GDK_IS_WAYLAND_DISPLAY (gdk_display_get_default ()))
+    {
+        GtkWindow *window;
+        GdkWindow *panelwin;
+        GtkWidget *toplevel;
+        int  x, y, w, h, panel_w, panel_h;
+
+        /*Get the calendar window dimensions*/
+        window = (GTK_WINDOW (cd->calendar_popup));
+        gtk_window_get_size(window, &w, &h);
```

Comment:
Can you please update code format? .......using a blank before a bracket?
This looks very wild in several places here :)

Reply:
No problem, will get on that now

Reply:
Done

Reply:
thx